pipelines/build-deploy-test-CI-WL.yml

```diff
@@ -45,6 +45,8 @@ groups:
 
 - name: "Whitelodge Deployments"
   jobs:
+  - "WL Terraform"
+  - "WL Rabbit Helm"
```

Comment:
Need to be added to the 'Overview' job as well

Reply:
Done